include/osquery/events.h

```diff
@@ -302,17 +302,15 @@ class EventPublisherPlugin : public Plugin,
 
  public:
   /// Number of Subscription%s watching this EventPublisher.
-  size_t numSubscriptions() const {
-    return subscriptions_.size();
-  }
+  size_t numSubscriptions();
 
   /**
    * @brief The number of events fired by this EventPublisher.
    *
    * @return The number of events.
    */
   EventContextID numEvents() const {
-    return next_ec_id_;
+    return next_ec_id_.load();
```

Comment:
While you're here, would you mind moving this implementation to the `.cpp` similar to `numSubscriptions`? I know it's trivial but we should keep implementations in the implementation file :\